integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkCommonIT.java

```diff
@@ -519,7 +496,8 @@ private void checkTableReadWrite(SparkTableInfo table) {
     Assertions.assertEquals(checkValues, queryResult.get(0));
   }
 
-  private String getCreateSimpleTableString(String tableName) {
+  // override this method in subclass to create table for different datasources
+  protected String getCreateSimpleTableString(String tableName) {
```

Comment:
why overide this method?

Reply:
I originally wanted to use 'using provider' to distinguish datasources from each other, but as you said, the corresponding table can already be created through catalog instance, so 'using provider' really can not be used.
I‘ll roll it back.

Reply:
updated.